src/components/PopupModal/PopupModal.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  PopupModal,
+  PopupModalContent,
+  PopupModalHeader,
+  PopupModalFooter,
+} from "./PopupModal";
+
+export default { title: "Components/PopupModal" };
+
+const args = {
+  onOpen: () => {
+    console.log("event on open");
+  },
+  onClose: () => {
+    console.log("event on close");
+  },
+};
+
+export const _Regular = {
+  render: () => (
+    <div className="h-[450px]">
```

Comment:
is this value used to be pixel perfect with design ?  Shouldn't we use `w-full` when we do our integrations ?